src/wifi_hal_nl80211_events.c

```diff
@@ -104,19 +103,9 @@ int notify_assoc_data(wifi_interface_info_t *interface, struct nlattr **tb,
         mgmt_frame.data = (unsigned char *)mgmt;
 #ifdef WIFI_HAL_VERSION_3_PHASE2
         callbacks->mgmt_frame_rx_callback(vap->vap_index, &mgmt_frame);
-#else
-#if defined(RDK_ONEWIFI) &&                                                                     \
-    (defined(TCXB7_PORT) || defined(CMXB7_PORT) || defined(TCXB8_PORT) || defined(XB10_PORT) || \
-        defined(TCHCBRV2_PORT) || defined(SCXER10_PORT))
-        if (tb[NL80211_ATTR_RX_PHY_RATE_INFO]) {
-            phy_rate = nla_get_u32(tb[NL80211_ATTR_RX_PHY_RATE_INFO]);
-        }
-        callbacks->mgmt_frame_rx_callback(vap->vap_index, sta_mac, (unsigned char *)mgmt, frame_len,
-            mgmt_type, dir, sig_dbm, phy_rate);
```

Comment:
Why are we removing the above change?

Reply:
this lines are causing below build issues.
``` ../git/src/wifi_hal_nl80211_events.c:61:9: error: unused variable 'phy_rate' [-Werror=unused-variable]
|    61 |     int phy_rate = 60;
|       |         ^~~~~~~~
| In file included from ../git/src/wifi_hal_nl80211_events.c:39:
| ../git/src/wifi_hal_nl80211_events.c: In function 'nl80211_new_station_event':
| ../git/src/wifi_hal_nl80211_events.c:161:24: error: format '%d' expects argument of type 'int', but argument 5 has type 'size_t' {aka 'long unsigned int'} [-Werror=format=]
|   161 |     wifi_hal_dbg_print("%s:%d: New station ies_len:%d, ies:%p\n", __func__, __LINE__, ies_len, ies);
|       |                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                      ~~~~~~~
|       |                                                                                       |
|       |                                                                                       size_t {aka long unsigned int}
| ../git/src/wifi_hal_priv.h:1049:81: note: in definition of macro 'wifi_hal_dbg_print'
|  1049 | #define wifi_hal_dbg_print(format, ...)  wifi_hal_print(WIFI_HAL_LOG_LVL_DEBUG, format, ##__VA_ARGS__)
|       |                                                                                 ^~~~~~
| ../git/src/wifi_hal_nl80211_events.c:161:53: note: format string is defined here
|   161 |     wifi_hal_dbg_print("%s:%d: New station ies_len:%d, ies:%p\n", __func__, __LINE__, ies_len, ies);
|       |                                                    ~^
|       |                                                     |
|       |                                                     int
|       |                                                    %ld
| ../git/src/wifi_hal_hostapd.c: In function 'update_security_config':
| ../git/src/wifi_hal_hostapd.c:667:25: error: 'struct hostapd_radius_servers' has no member named 'fallback_already_done'
|   667 |             conf->radius->fallback_already_done = false;
|       |                         ^~
| ../git/src/wifi_hal_nl80211_events.c: In function 'notify_assoc_data':
| ../git/src/wifi_hal_nl80211_events.c:132:1: error: control reaches end of non-void function [-Werror=return-type]
|   132 | }
|       | ^
| make[2]: *** [Makefile:1120: librdk_wifihal_la-wifi_hal_hostapd.lo] Error 1
| make[2]: *** Waiting for unfinished jobs....
| ../git/src/wifi_hal_nl80211.c:14733:6: error: 'const struct wpa_driver_ops' has no member named 'radius_fallback_failover'
| 14733 |     .radius_fallback_failover = wifi_drv_send_radius_fallback_and_failover,
|       |      ^~~~~~~~~~~~~~~~~~~~~~~~
| ../git/src/wifi_hal_nl80211.c:14733:33: error: excess elements in struct initializer [-Werror]
| 14733 |     .radius_fallback_failover = wifi_drv_send_radius_fallback_and_failover,
|       |                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
| ../git/src/wifi_hal_nl80211.c:14733:33: note: (near initialization for 'g_wpa_driver_nl80211_ops')
| cc1: all warnings being treated as errors
| make[2]: *** [Makefile:1127: librdk_wifihal_la-wifi_hal_nl80211.lo] Error 1 

Also, this lines are present  in notify_assoc_data() . This function is needed only for reference platforms but removed lines are defined under "#if defined(RDK_ONEWIFI) &&                                                                     \
    (defined(TCXB7_PORT) || defined(CMXB7_PORT) || defined(TCXB8_PORT) || defined(XB10_PORT) || \
        defined(TCHCBRV2_PORT) || defined(SCXER10_PORT))"